src/cirrus/builtins/cloudformation/resources.yml

```diff
@@ -19,7 +19,7 @@ Resources:
     Type: AWS::SQS::Queue
     Properties:
       QueueName: "#{AWS::StackName}-process"
-      VisibilityTimeout: 1000
+      VisibilityTimeout: 20
```

Comment:
Found something interesting about this setting while looking for something else in the AWS docs: https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#events-sqs-queueconfig.

Appears they recommend we set this to 6x the lambda timeout, which is 18s. So this should be 90s? I suppose that doesn't fail quite as fast as I hoped (for DLQ alerts), but it still isn't too bad.

Reply:
I don't know where 90s came from. It should be 108s.

That makes the time from message receipt to a message falling into the DLQ roughly 9 minutes. Add additional time for alerting to fire, where applicable, and this is quite a long time.

I think we should also reduce the timeout on the `process` lambda itself to help get this time down. A value of 10s, for example make this visibility timeout 60s, providing a time-to-first-alert of 5-6 minutes, which I think is much more reasonable in cases with short SLAs. Perhaps I will go with these values for now but also make a follow up ticket to benchmark or analyze past `process` timing, to see if we should recommend reducing this value even further.

Reply:
Created https://github.com/cirrus-geo/cirrus-geo/issues/167 as suggested.